6917663: test/java/security/Provider/Turkish.java not samevm friendly
Summary: Added othervm flag to ensure that this test will run in isolation.
Reviewed-by: alanb

--- openjdk.orig/jdk/test/java/security/Provider/Turkish.java        2010-01-18 11:02:18.000000000 +0100
+++ openjdk/jdk/test/java/security/Provider/Turkish.java        2010-01-18 11:02:17.000000000 +0100
@@ -25,6 +25,7 @@
  * @test
  * @bug 6220064
  * @summary make sure everything works ok in the Turkish local (dotted/dotless i problem)
+ * @run main/othervm Turkish
  * @author Andreas Sterbenz
  */

